Data/Csv/Conversion.hs

```diff
@@ -798,6 +799,22 @@ instance FromField () where
     parseField _ = pure ()
     {-# INLINE parseField #-}
 
+instance FromField a => FromField (Identity a) where
+    parseField = fmap Identity . parseField
+    {-# INLINE parseField #-}
+
+instance ToField a => ToField (Identity a) where
+    toField = toField . runIdentity
+    {-# INLINE toField #-}
+
+instance FromField a => FromField (Const a b) where
```

Comment:
Ok, what I'm missing here (besides a `@since 0.5.2` annotation), is describing somewhere what the pratical use-case for this instance is.

Reply:
If there exists a unique possible implementation of an instance, it should be defined. My actual motivation is to use `extensible`'s record because they are parametrised by wrapper types (Usually Identity).